### ISSUE TITLE
How to get complete AuthResponse object from google signin? Not able to fetch expires_in property from response.   

### ISSUE THREAD

**Issue by bhorpooja** [abacritt/angularx-social-login#365]
I'm facing problem to get  expires_in attribute from  this.socialAuthService.signIn() method, it does not exist in response.

Angular Version: 7.0.0

Need to update below highlighted code in GoogleLoginProvider to get all property

`angularx-social-login\angularx-social-login.es5.js`

File code: 
getLoginStatus() and signIn() methods to be update for GoogleLoginProvider

```
`   /**
     * @return {?}
     */
    GoogleLoginProvider.prototype.getLoginStatus = /**
     * @return {?}
     */
    function () {
        var _this = this;
        return new Promise(function (resolve, reject) {
            _this.onReady().then(function () {
                if (_this.auth2.isSignedIn.get()) {
                    var /** @type {?} */ user = new SocialUser();
                    var /** @type {?} */ profile = _this.auth2.currentUser.get().getBasicProfile();
                    **var /** @type {?} */ authRes = _this.auth2.currentUser.get().getAuthResponse();**
                    var /** @type {?} */ token = _this.auth2.currentUser.get().getAuthResponse(true).access_token;
                    var /** @type {?} */ backendToken = _this.auth2.currentUser.get().getAuthResponse(true).id_token;
                    user.id = profile.getId();
                    user.name = profile.getName();
                    user.email = profile.getEmail();
                    user.photoUrl = profile.getImageUrl();
                    user.firstName = profile.getGivenName();
                    user.lastName = profile.getFamilyName();
                    **user.authResponse = authRes;**
                    user.authToken = token;
                    user.idToken = backendToken;
                    resolve(user);
                }
            });
        });
    };`

```
Can some please look into this?

**Comment by jaibatrik** [comment]
Yeah, this needs a fix. Feel free to send a PR. Else, I'll make the change when I get some time.

**Comment by bhorpooja** [comment]
Hi @jaibatrik,

I have created PR. Can you please look into it

**Comment by jaibatrik** [comment]
Not remembering if I released this fix to npm. @bhorpooja did you receive the fix?

**Comment by bhorpooja** [comment]
> Not remembering if I released this fix to npm. @bhorpooja did you receive the fix?

@jaibatrik  No. I didn't receive any fixes.

**Comment by jaibatrik** [comment]
Published to npm. Fix should be available at v3.5.5.
